### PR TITLE
Feature/25/appointment booking form

### PR DESCRIPTION
**Pull request by Alan-day**
We created a desktop and mobile booking form. Form creates an object that is logged to a console. At the moment booking data is not assigned to a variable or passed to mockData.js.

### CODE REVIEW

File: src/pages/Resources/Resources.scss [hunk 1]
```diff
@@ -7,6 +7,7 @@
 
   .desktop__container {
     display: flex;
+    height: 100vh;
```
Comment (olirob93) [inline src/pages/Resources/Resources.scss:10]: I removed this, not sure we need this why did you add it back in?

File: src/pages/Clients/Clients.scss [hunk 2]
```diff
@@ -3,6 +3,7 @@
 
 .clients-container {
   background-color: #ebebf5;
+  height: 100vh;
```
Comment (olirob93) [inline src/pages/Clients/Clients.scss:6]: Not sure we need this either?

File: src/pages/Booking/Booking.scss [hunk 3]
```diff
@@ -0,0 +1,29 @@
+@use "../../styles/variables.scss" as *;
+
+.booking {
+  display: flex;
+  background-color: #ebebf5;
+  height: 100vh;
```
Comment (olirob93) [inline src/pages/Booking/Booking.scss:6]: You need to change the height on mobile as it's got white space at the bottom

File: src/components/Forms/AppointmentBookingForm/AppointmentBookingForm.jsx
Comment (olirob93) [inline src/components/Forms/AppointmentBookingForm/AppointmentBookingForm.jsx:1]: I think this needs work, On desktop for me the buttons at the bottom overflow at the bottom of the page, and on the screen size 1024 which is a small laptop is a bit broken. See image attached
Screenshot 2023-05-11 110330

**Review by olirob93 — CHANGES_REQUESTED** [review]
Quite a few changes are needed on this one. It's not totally responsive but the functionality looks good.

File: src/pages/Resources/Resources.scss (hunk 1 above)
Comment (Alan-day) [reply src/pages/Resources/Resources.scss:10]: When we remove height:100vh the mobile version can't be scrolled

File: src/pages/Clients/Clients.scss (hunk 2 above)
Comment (Alan-day) [reply src/pages/Clients/Clients.scss:6]: Ok, deleted.

File: src/components/Forms/AppointmentBookingForm/AppointmentBookingForm.jsx
Comment (Alan-day) [reply src/components/Forms/AppointmentBookingForm/AppointmentBookingForm.jsx:1]: Added new breakpoint that changes margin, size and font of the calendar when it goes over mobile breakpoint, laptop version looks ok now.

File: src/pages/Booking/Booking.scss (hunk 3 above)
Comment (Alan-day) [reply src/pages/Booking/Booking.scss:6]: I think we need it for displaying background colour on the whole mobile page

**Review by olirob93 — APPROVED** [review]
still a few things but i'll fix it on my PR so we can get this one over the line
